aws/logs_monitoring/tools/integration_tests/integration_tests.sh

```diff
@@ -215,6 +215,7 @@ if [ $ADDITIONAL_LAMBDA == true ]; then
 		done
 
 		if [ "$mismatch_found" = true ]; then
+		    sls remove
```

Comment:
Why do we need to remove the stack here?

Reply:
Before exiting, just for cleanup.